ui/src/containers/VolumeInformation.js

```diff
@@ -73,10 +73,10 @@ const VolumeInformation = props => {
           activeColor={theme.brand.secondary}
           paths={[
             <StyledLink to="/nodes">{intl.messages.nodes}</StyledLink>,
-            <StyledLink to={`/nodes/${node.name}/volumes`}>
+            <StyledLink to={`/nodes/${node.name}/volumes`} title={node.name}>
               {node.name}
             </StyledLink>,
-            <BreadcrumbLabel>{match.params.volumeName}</BreadcrumbLabel>
+            <BreadcrumbLabel>{match.params.volumeName}</BreadcrumbLabel>,
```

Comment:
need ```title``` here too? I think you should add it to every items of the breadcrumbs

Reply:
Thanks. Going to check with core-ui,  because the last item of not <a> we don't have `title` for that